src/util/pup_stl.h

```diff
@@ -127,7 +128,9 @@ namespace PUP {
     size_t nChar=v.length();
     p|nChar;
     if (p.isUnpacking()) { //Unpack to temporary buffer
+      CmiEnforce(nChar <= std::numeric_limits<size_t>::max());
```

Comment:
Somehow LGTM _did_ consider this to fix the alert, even though the malloc above did not work. This confuses me, but we can leave it.